parity/cache.rs

```diff
@@ -52,7 +52,9 @@ impl Default for CacheConfig {
 }
 
 impl CacheConfig {
-	/// Creates new cache config with cumulative size equal `total`.
+	/// Creates new cache config with cumulative size equal to `total`, distributed as follows: 70%
+	/// to rocksdb, 10% to the blockchain cache and 20% to the state cache. The transaction queue
+	/// cache size is set to 40Mb and the trace cache to 20Mb.
```

Comment:
I wonder if we should reconsider the defaults

Reply:
You might be right. Or not. Really hard to even have an opinion about this without some serious testing.